<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMonthDialog</class>
 <widget class="QDialog" name="NewMonthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decision</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/question_mark.png</normaloff>:/img/question_mark.png</iconset>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>181</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Sum Brown Coal</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Sum Black Coal</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="labelBrownMinedMixed">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelBrownCoalMined">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Black Coal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Brown Coal</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelBlackMinedMixed">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelBlackCoalMined">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Coal mined this month</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelSumBlack">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="labelSumBrown">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Mixed -&gt; Black</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Mixed -&gt; Brown</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Miners</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelMoneyDialog">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Money</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSeason">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Season</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>251</width>
     <height>356</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="15" column="1">
     <widget class="QLabel" name="labelCashHeatingPlant">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="labelNeedsBlackHeatingPlant">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Cash / unit</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Needs (brown)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Power Station</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="labelCashCoalStorage">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Needs (black)</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="labelNeedsBrownHeatingPlant">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelCashIronWorks">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Cash / unit</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Coal Storage Site</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="labelNeedsBrownCoalStorageSite">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Needs (black)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Needs (black)</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Cash / unit</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Needs (black)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="labelNeedsBlackPowerstation">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Needs (brown)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Ironworks</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="labelCashPowerstation">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="labelNeedsBrownPowerstation">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Heating Plant</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelNeedsBlackCoalStorage">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Cash / unit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelNeedsBrownIronworks">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Needs (brown)</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Needs (brown)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelNeedsBlackIronworks">
      <property name="text">
       <string>No data</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QLineEdit" name="lineEditBlackHeatingPlant"/>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="9" column="2">
     <widget class="QLineEdit" name="lineEditBlackPowerStation"/>
    </item>
    <item row="14" column="2">
     <widget class="QLineEdit" name="lineEditBrownHeatingPlant"/>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="lineEditBrownCoalStorageSite"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Amount to sell</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEditBlackCoalStorageSite"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditBlackIronWorks"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEditBrownIronWorks"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="storageLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_31">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Storage</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
